### ISSUE TITLE
fsync parent also after rename

### ISSUE THREAD

**Issue by radumarias** [andreacorbellini/rust-atomic-write-file#8]
After reading more I found it's recommended to fsync parent also to be sure data is persisted ok.
You can see here more https://www.quora.com/Linux/When-should-you-fsync-the-containing-directory-in-addition-to-the-file-itself
Also I see other implementations are doing it https://github.com/untitaker/rust-atomicwrites/blob/master/src/lib.rs#L200

It's not a bit deal, I can fsync parent after commit but would be nice to have this included

**Comment by andreacorbellini** [comment]
Hi, yes, this change has already been made and will be part of the next
release:
https://github.com/andreacorbellini/rust-atomic-write-file/commit/7da14cbe37c011f3f9968a78015288ea60cd04e1

For context: when I first made this crate, I thought that it would be the
caller responsibility to sync the directory if they wanted to (there are
some cases where they might not want to; for example when writing multiple
files, in which case it makes sense to sync the directory only once at the
end). And so I omitted that call to save a syscall. However, I later
realized that callers might not be aware or want to deal with such low
level details, and so I added the call.

Thanks for reporting this. I'll close the issue once the release is made.

**Comment by radumarias** [comment]
sure thing, great job 👍

**Comment by Luflosi** [comment]
I think this can be closed as https://github.com/andreacorbellini/rust-atomic-write-file/releases/tag/v0.2.0 has been released.
